Doc/howto/logging-cookbook.rst

```diff
@@ -1675,10 +1675,10 @@ That can still use %-formatting, as shown here::
     >>>
 
 Logging calls (``logger.debug()``, ``logger.info()`` etc.) only take
-positional parameters for the actual logging message itself, with keyword
-parameters used only for determining options for how to handle the actual
-logging call (e.g. the ``exc_info`` keyword parameter to indicate that
-traceback information should be logged, or the ``extra`` keyword parameter
+positional arguments for the actual logging message itself, with keyword
```

Comment:
```suggestion
positional (or keyword) arguments for the logging message itself, with keyword
```
'actual' and 'itself' are redundant.

Reply:
"(or keyword)" is not correct, as explained further in this sentence.
```suggestion
positional arguments for the logging message itself, with keyword
```

Reply:
Call with `msg='log message'` works, but `msg='error %s', args=('Syntax',)` does not, as explicit args conflicts with *args collecting non-existent positional args as `()`.  So I agree with not mentioning the `msg`-only by keyword possibility.